### PR TITLE
Fix bar elements

### PR DESCRIPTION
**Pull request by ptbrowne**
Before

image

After

image

### CODE REVIEW

File: src/styles/base.css [hunk 1]
```diff
@@ -42,3 +41,8 @@
 [role=banner] [data-icon='icon-claudy'] {
   background-image: url(../assets/icons/16/icon-claudy.svg)
```
Comment (CPatchane) [inline src/styles/base.css]: Missing `;`

Comment (ptbrowne) [reply src/styles/base.css]: ? `;` are missing everywhere in this file. It does not matter, it is on the last line.

Comment (ptbrowne) [reply src/styles/base.css]: Anyway, I added them in https://github.com/cozy/cozy-bar/pull/133/commits/a22bfd1d8d721a9d4470db35afc3ffcc6a4ae8b9

**Comment by y-lohse** [comment]
Please ignore, I just need a url for this image.

cozy-bar-triplet

**Pull request merged**